scripts/lsp_types.lua

```diff
@@ -184,17 +185,21 @@ function M.gen(opt)
     output[#output + 1] = line
   end
 
-  tofile(output_file, table.concat(output, '\n'))
+  tofile(opt.output_file, table.concat(output, '\n'))
 end
 
-local opt = {}
-
-local index = 1
-for _, a in ipairs(arg) do
-  if vim.startswith(a, '--') then
-    local name = a:sub(3)
-    opt[index] = name
-    index = index + 1
+local opt = {
+  output_file = 'runtime/lua/vim/lsp/types/protocol.lua',
+  ref = "main"
```

Comment:
Is this script not covered by our linter?

```suggestion
  ref = 'main',
```